Source/CarthageKit/GitHub.swift

```diff
@@ -118,11 +118,14 @@ public struct GitHubRelease: Equatable {
 	public let ID: Int
 
 	/// The name of this release.
-	public let name: String
+	public let name: String?
 
-	/// The name of this release, with fallback to its tag when the name is an empty string.
+	/// The name of this release, with fallback to its tag when the name is an empty string or nil.
 	public var nameWithFallback: String {
-		return name.isEmpty ? tag : name
+		if let name = name {
+			return name.isEmpty ? tag : name
+		}
+		return tag
```

Comment:
I don't like the duplication of `tag` here. Could we use a signal conditional here so that it doesn't have to be repeated? :pray:

You should be able to use [Swift 1.2's improved optional binding](http://nshipster.com/swift-1.2/).